scripts/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
I've generally preferred `/usr/bin/env bash` for scripts like this, is there a benefit to hardcoding `/bin/sh`?

Reply:
Mostly habit, though I'd definitely use `sh` over `bash`.

`sh` is asking for a POSIX compatible shell, which may or not happen to be `bash`. The former is _usually_ faster to start. Notably about five years ago Ubuntu (and maybe other Debian systems) sped up their boot time quite a lot by changing all their boot scripts to `#!/bin/sh` over `#!/bin/bash`.

`/bin/sh` is pretty close to guaranteed to be present in a Linux system. I'm not sure why that path should be any less likely to be present than `/usr/bin/env` (which is more common for launching other interpreters), so I don't find it compelling to launch two processes to run a script (`env` then the shell) over just the one which the current spelling does.